controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -236,15 +239,45 @@ void wfa_ca::handle_wfa_ca_message(
 
     switch (command_type) {
     case eWfaCaCommand::CA_GET_VERSION: {
-        // TODO
+
+        /* 
+        * This command is used to obtain the version of the Control Agent software. The command does
```

Comment:
This should be indented one space more to align the asterisks.

I've fixed this up while rebasing.